functions/completed-checks/check-form.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
```

Comment:
I would prefer to see this file (as well as any others) identified as test data if that is their purpose.  We could either put them in a `test` directory or mark them in the filename.

Reply:
@jon-shipley moved to mocks folder within the function's directory.